regolith/helpers/a_expensehelper.py

```diff
@@ -0,0 +1,184 @@
+"""
+Helper to add expenses.
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+)
+
+TARGET_COLL = "projecta"
+ALLOWED_TYPES = ["business", "travel"] # need to check all expense types.
+ALLOWED_STATI = ["submitted", "unsubmitted", "reimbursed"]
+
+
+def subparser(subpi):
+    subpi.add_argument("amount", help="expense amount",
+                       )
+    subpi.add_argument("name", help="A short name for the expense",
+                       default=None
+                       )
+    subpi.add_argument("-b", "--business",
```

Comment:
you need an ``action='store_true'``.  Please see https://docs.python.org/2/library/argparse.html

Reply:
Thank you. I will fix that